plugins/out_stackdriver/stackdriver.c

```diff
@@ -2467,41 +2491,46 @@ static int stackdriver_format_test(struct flb_config *config,
     return 0;
 
 }
-
 #ifdef FLB_HAVE_METRICS
-static void update_http_metrics(struct flb_stackdriver *ctx,
-                                struct flb_event_chunk *event_chunk,
-                                uint64_t ts,
-                                int http_status)
-{
-    char tmp[32];
+static void add_record_metrics(struct flb_stackdriver* ctx,
+                               uint64_t ts,
+                               int val,
+                               int response_code) {
+    char response_code_label[32];
 
     /* convert status to string format */
-    snprintf(tmp, sizeof(tmp) - 1, "%i", http_status);
-    char *name = (char *) flb_output_name(ctx->ins);
+    snprintf(response_code_label, sizeof(response_code_label) - 1, "%i",
+             response_code);
+    char* name = (char*) flb_output_name(ctx->ins);
```

Comment:
There should be no declarations in the middle of the function; should always be at the top